### PR TITLE
Add extra properties for userProfile

### PR DESCRIPTION
**Pull request by vedi**

### CODE REVIEW

**Comment by refaelos** [comment]
Do we still need this PR ?

**Comment by vedi** [comment]
it's implemented here: https://github.com/soomla/android-profile/pull/11

File: SoomlaAndroidProfile/.idea/android-profile.iml [hunk 1]
```diff
@@ -1,24 +1,14 @@
...
     <content url="file://$MODULE_DIR$">
-      <sourceFolder url="file://$MODULE_DIR$/.idea/gen" isTestSource="false" generated="true" />
       <sourceFolder url="file://$MODULE_DIR$/src" isTestSource="false" />
     </content>
     <orderEntry type="inheritedJdk" />
     <orderEntry type="sourceFolder" forTests="false" />
-    <orderEntry type="module" module-name="facebook" />
-    <orderEntry type="module-library">
+    <orderEntry type="module" module-name="facebook" scope="PROVIDED" />
+    <orderEntry type="module-library" scope="PROVIDED">
```
Comment (vedi) [inline SoomlaAndroidProfile/.idea/android-profile.iml]: Why did you do this?

File: SoomlaAndroidProfile/res/values/strings.xml [hunk 2]
```diff
@@ -1,4 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<resources>
-    <string name="app_name">Android Profile</string>
-</resources>
```
Comment (vedi) [inline SoomlaAndroidProfile/res/values/strings.xml]: Why did you remove this?
